### DISCUSSION TITLE
What would you like to learn?

### DISCUSSION THREAD

**Discussion by timothywarner** [timothywarner/copilotaio#5] [Ideas]
Hey there! Please feel free to post in here to let me know what particular GitHub Copilot-related content you'd like me to include in my training. Thanks, Tim

**Comment by timothywarner** [comment]
One idea I have is a 2-4 hour end-to-end training on Microsoft Entra hybrid identity strategies. From design to implementation to troubleshooting. Thoughts?

**Comment by alextaylor-the-dev** [comment]
I'd like to know how to convert OpenAI python scripts into Azure OpenAI Service pls
